.eslintrc.js

```diff
@@ -179,6 +179,44 @@ module.exports = {
         "@typescript-eslint/explicit-module-boundary-types": "off",
       },
     },
+    {
+      files: "*.mdx",
+      extends: ["plugin:mdx/recommended"],
+      rules: {
+        "react/jsx-filename-extension": ["error", { extensions: [".mdx"] }],
+        "mdx/no-unescaped-entities": "OFF",
```

Comment:
I also suggest setting `mdx/remark` to `"error"` ("recommended" config sets it to `"warn"`) because we want to prevent code with violations from being merged.

Reply:
If it's set globally like that, I'd probably not use the prose linting that way. With Vale, we can set the error level for individual rules (some prose rules are more reliable than others, I've found).

Reply:
Sounds good 👍